src/flatMap.js

```diff
@@ -1,3 +1,3 @@
 export function flatMap(callback, thisArg) {
```

Comment:
I think we can drop `thisArg` and keep `callback` so that `Array.prototype.flatMap.length` matches the native implementation.

Reply:
Fixed